manage.py

```diff
@@ -0,0 +1,31 @@
+from manager import Manager
+from quadcore.crawler.rss import RSSCrawler
+from quadcore.crawler.github import GithubCrawler
+
+import quadcore.tests
+import json
+import sys
+import unittest
+
+manager = Manager()
+
+@manager.command
+def test(category):
```

Comment:
let's call this manual_run? "test" typically means unit testing or integration testing.

Reply:
I'll fix, commit, and merge it with this PR.